README.md

```diff
@@ -1 +1,976 @@
+SUPERVISOR VERSION: 9.15.7
+IS WEB ACCESSIBLE:  false
+OS VERSION:         balenaOS 2.38.0+rev1
+DASHBOARD URL:      https://dashboard.balena-cloud.com/devices/76543217654321765432176543217654/summary
+```
+As we now have the local IP address for it, we can use this to call the command
+to leave the balenaCloud environment:
+```
+$ BALENARC_DATA_DIRECTORY=. balena leave 192.168.1.171
+[Success] Device successfully left the platform.
+```
+The device now becomes *unmanaged*. This means it acts in the same way as
+a device that has been provisioned with an unconfigured balenaOS image, for
+example one that has been downloaded from
+[https://www.balena.io/os/#download](https://www.balena.io/os/#download).
+
+We can now join a different balena environment by using balenaCLI to login
+to it. As we previously did this for our staging environment, we can simply
+use the data and tokens we saved for this by changing directories and using
+the other environment information:
+```
+$ cd ~/balenaStaging
+```
+We'll now create a new application on the staging environment to move the
+device to:
+```
+$ BALENARC_DATA_DIRECTORY=. balena app create stagingCliApp --type fincm3
+Application created: stagingCliApp (fincm3, id 97654)
+```
+Finally, we'll issue a command to the now unmanaged device to join the staging
+environment and the `stagingCliApp` application:
+```
+$ BALENARC_DATA_DIRECTORY=. balena join 192.168.1.171 --application stagingCliApp
+? Check for updates every X minutes 10
+[Success] Device successfully joined balena-staging.com!
+```
+We can now check the devices on the staging environment to ensure it's joined
+successfully:
+```
+$ BALENARC_DATA_DIRECTORY=. balena apps
+BALENARC_DATA_DIRECTORY=. balena devices
+ID     UUID    DEVICE NAME      DEVICE TYPE  APPLICATION NAME  STATUS IS ONLINE SUPERVISOR VERSION OS VERSION           DASHBOARD URL
+876542 3456789 purple-snowflake fincm3       stagingCliApp     Idle   true      9.15.7             balenaOS 2.38.0+rev1 https://dashboard.balena-staging.com/devices/3456789345678934567893456789/summary
+```
+As can be seen, it's been given a new name and UUID.
+
+If we hadn't specified the application to join, we would have seen an
+interactive list of all the applications on the staging environment:
+```
+$ BALENARC_DATA_DIRECTORY=. balena join 192.168.1.171
+? Select application
+❯ heds/artik530
+  heds/orangepi
+  heds/stagingcliapp
+  heds/switchapp
+  heds/vpn-changed
+```
+Note that the applications listed are not all of the `fincm3` device type, but
+share a common architecture, so this could be `aarch64`, `armv7hf` or `armv6`
+for the Fin. You should be very careful when selecting an application of a
+different device type, as you may find issues when the device attempts to run
+the application code for a device without the same peripherals or system-on-chip
+layout.
+
+### 2.2.2 Production Devices
+
+>>> THIS WILL NOT WORK FOR A PRODUCTION DEVICE, BECAUSE AS SOON AS YOU USE
```

Comment:
maybe worth linking to any relevant issue(s) in all bug sections?

Reply:
Yup

Reply:
Done